topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -2,13 +2,16 @@ package strings
 
 func ReverseString(str string) string {
 
-	// Create a slice of runes.
+	// Convert the input string into slice of runes for processing.
+	// A runes are unicode code-points.
 	runes := []rune(str)
 
-	// Create an index that will traverse the collection of runes from the beginning to the end.
+	// Create an index that will traverse the collection of
+	// runes from the beginning to the end.
 	var beg int
 
-	// Create an index that will traverse the collection of runes from the end to the beginning.
+	// Create an index that will traverse the collection of
+	// runes from the end to the beginning.
 	end := len(runes) - 1
 
 	// Keep swapping runes until the two indexes meet in the middle.
```

Comment:
Read the comment about swapping code points. We are now saying runes.

Reply:
Ok.